core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -73,7 +73,8 @@ object ApiVersion {
     // Introduced OffsetsForLeaderEpochRequest V1 via KIP-279
     KAFKA_2_0_IV0,
     // Introduced ApiVersionsRequest V2 via KIP-219
-    KAFKA_2_0_IV1
+    KAFKA_2_0_IV1,
+    KAFKA_2_0_IV2
```

Comment:
I think it's legitimate to bump the internal broker version for the schema change for offset/group metadata. Perhaps we can mention this here? Also, we should probably use create a new version for Kafka 2.1.